python/AI/AIWrapper.cpp

```diff
@@ -202,7 +205,8 @@ namespace FreeOrionPython {
         def("updateResearchQueue",                  AIInterface::UpdateResearchQueue);
         def("updateProductionQueue",                AIInterface::UpdateProductionQueue);
 
-        def("issueFleetMoveOrder",                  IssueFleetMoveOrder, "Orders the fleet with indicated fleetID (int) to move to the system with the indicated destinationID (int). Returns 1 (int) on success or 0 (int) on failure due to not finding the indicated fleet or system.");
+        def("issueFleetMoveOrder",                  IssueFleetMoveOrder2, "Orders the fleet with indicated fleetID (int) to move to the system with the indicated destinationID (int). Returns 1 (int) on success or 0 (int) on failure due to not finding the indicated fleet or system.");
+        def("issueFleetMoveOrder",                  IssueFleetMoveOrder3, "Orders the fleet with indicated fleetID (int) to move to the system with the indicated destinationID (int). Append (true) the destination to an already present route, or set the destination as route (false). Returns 1 (int) on success or 0 (int) on failure due to not finding the indicated fleet or system.");
```

Comment:
Can we just name it `appendFleetMoveOrder`?

Reply:
I now renamed the methods. My AI still works as expected (after adjusting the method call).